src/test/java/jenkins/plugins/office365connector/ValidateUrlTest.java

```diff
@@ -0,0 +1,38 @@
+package jenkins.plugins.office365connector;
+
+import java.util.Arrays;
+import java.util.Collection;
+
+import jenkins.plugins.office365connector.util.FormUtils;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+@RunWith(Parameterized.class)
+public class ValidateUrlTest {
+
+    @Parameterized.Parameter()
+    public String input;
+    @Parameterized.Parameter(value = 1)
+    public boolean expected;
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
```

Comment:
duplicated code?

Reply:
Always remove for refactoring, and squashing later